src/test/java/uk/gov/hmcts/reform/idam/util/SecurityUtilTest.java

```diff
@@ -26,17 +27,27 @@ class SecurityUtilTest {
     @Mock
     private IdamClient idamClient;
 
+
     private static final String TOKEN = "123456789";
     private static final String ACCESS_TOKEN = "Bearer 1234";
+    public static final String IDAM_TOKEN = "1234";
+    public static final String CLIENT_CREDENTIALS_GRANT = "client_credentials";
+    public static final String SCOPE = "archive-user view-archived-user delete-archived-user";
 
     @InjectMocks
     private SecurityUtil securityUtil;
 
+    @InjectMocks
+    private TokenResponse tokenResponse;
+
```

Comment:
I don't think this being used anywhere and it can be safely deleted. In general if you inject mocks into multiple things, that might be an indication that it is no longer a unit test.

Reply:
This has been removed. Thank you for clarification